galaxykubeman/values.yaml

```diff
@@ -53,3 +53,6 @@ galaxy:
   #     configMapKeyRef:
   #       name: "{{.Release.Name}}-[galaxykubeman|leo]-configs"
   #       key: WORKSPACE_NAME
+
+rbac:
```

Comment:
This should be `rbac: {}` since all of its keys will be commented out in the default values and expecting it to be set at install.

Reply:
I just pushed adding `enabled: true`, so does this comment still apply?

Reply:
Nope, but in that case, the top should be `{{- if .Values.rbac.enabled }}`

Reply:
ahhh good point. will update!